samples/application-insights/sample.md

```diff
@@ -36,21 +38,25 @@ By default the sample code loads this key from an environment variable called `A
 
 snippet: configure-ai-instrumentation-key
 
+The instrumentation key can be obtained under the Application Insights instance properties pane in the Azure portal.
```

Comment:
I'll have to push back on this. The sample is not focusing on teaching how to use AI. Documenting anything on the portal exactly to the clicks is insane. And it will break fast 😉